### PR TITLE
KEB does not configure zones for SKRs

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
- KEB must not configure multiple zones as a default setting

**Related issue(s)**
Related: https://github.tools.sap/kyma/backlog/issues/725

### CODE REVIEW

**Comment by crabtree** [comment]
Please bump the component image.

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ These are the provisioning parameters for Azure that you can configure:
 | **machineType** | string | Specifies the provider-specific virtual machine type. | No | `Standard_D8_v3` |
 | **volumeSizeGb** | int | Specifies the size of the root volume. | No | `50` |
 | **region** | string | Defines the cluster region. | No | `westeurope` |
-| **zones** | string | Defines the list of zones in which the Runtime Provisioner creates the cluster. | No | `["1", "2", "3"]` |
+| **zones** | string | Defines the list of zones in which the Runtime Provisioner creates the cluster. | No | `[]` |
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-service-description.md:49]:
```suggestion
| **zones** | string | Defines the list of zones in which Runtime Provisioner creates a cluster. | No | `[]` |
```

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 2]
```diff
@@ -64,7 +64,7 @@ These are the provisioning parameters for Azure that you can configure:
 | **machineType** | string | Specifies the provider-specific virtual machine type. | No | `Standard_D4_v3` |
 | **volumeSizeGb** | int | Specifies the size of the root volume. | No | `50` |
 | **region** | string | Defines the cluster region. | No | `westeurope` |
-| **zones** | string | Defines the list of zones in which the Runtime Provisioner creates the cluster. | No | `["1", "2", "3"]` |
+| **zones** | string | Defines the list of zones in which the Runtime Provisioner creates the cluster. | No | `[]` |
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-service-description.md:67]:
```suggestion
| **zones** | string | Defines the list of zones in which Runtime Provisioner creates a cluster. | No | `[]` |
```

File: docs/kyma-environment-broker/03-01-service-description.md [hunk 3]
```diff
@@ -87,7 +87,7 @@ These are the provisioning parameters for GCP that you can configure:
 | **machineType** | string | Specifies the provider-specific virtual machine type. | No | `n1-standard-4` |
 | **volumeSizeGb** | int | Specifies the size of the root volume. | No | `30` |
 | **region** | string | Defines the cluster region. | No | `europe-west4` |
-| **zones** | string | Defines the list of zones in which the Runtime Provisioner creates the cluster. | No | `["a", "b", "c"]` |
+| **zones** | string | Defines the list of zones in which the Runtime Provisioner creates the cluster. | No | `[]` |
```
Comment (klaudiagrz) [inline docs/kyma-environment-broker/03-01-service-description.md:90]:
```suggestion
| **zones** | string | Defines the list of zones in which Runtime Provisioner creates a cluster. | No | `[]` |
```

**Review by klaudiagrz — APPROVED** [review]

**Review by crabtree — APPROVED** [review]

**Pull request merged**